java/clients/src/main/java/sleeper/clients/QueryWebSocketClient.java

```diff
@@ -59,6 +66,7 @@ public class QueryWebSocketClient {
     private final String apiUrl;
     private final Client client;
     private final ConsoleOutput out;
+    private Instant startTime;
```

Comment:
It looks like this field can be a local variable?

Reply:
Done, thanks